regulations/views/reader.py

```diff
@@ -39,6 +39,7 @@ def get_context_data(self, **kwargs):
             'toc':          toc,
             'parts':        parts,
             'versions':     versions,
+            'left_sidebar_content': 'regulations/partials/left_sidebar_views/subpart_view.html',
```

Comment:
A custom left sidebar for the search view will be more clear and allow all of this logic to remain at the template level.

Reply:
Implemented in d695d5a